code/datums/uplink/devices and tools.dm

```diff
@@ -9,6 +9,13 @@
 	item_cost = 5
 	path = /obj/item/weapon/storage/toolbox/syndicate
 	desc = "Danger. Very robust. Filled with advanced tools."
+
+/datum/uplink_item/item/tools/gun_mods
+	name = "Gun Mods Kit"
+	item_cost = 20
+	path = /obj/item/weapon/storage/box/syndie_kit/gun_mods
+	desc = "Containe all gun mods that knewed to humanity."
```

Comment:
Gun mods should be provided individually, not like this.

Reply:
Removed